src/main/java/io/github/mfvanek/pg/model/DuplicatedIndexes.java

```diff
@@ -74,7 +117,9 @@ public String toString() {
     }
 
     public static DuplicatedIndexes of(@Nonnull final List<IndexWithSize> duplicatedIndexes) {
-        return new DuplicatedIndexes(duplicatedIndexes);
+        return new DuplicatedIndexes(duplicatedIndexes.stream()
+                .sorted(INDEX_WITH_SIZE_COMPARATOR)
```

Comment:
Consider moving sorting to the constructor. There would be no need for sorting in all of the factory methods. Also, that way `indexesNames` can be collected without additional sorting.

Reply:
Fixed